### PR TITLE
[Fix] Feedback Timmi

### PR DESCRIPTION
**Pull request by RipripKipkip**
chrome_QTUxnMjINC
chrome_TpKfkafkxN

image

### CODE REVIEW

File: packages/scss/src/components/_nav-side.scss [hunk 1]
```diff
@@ -283,7 +283,6 @@
 			color: _component("navSide.compact-palette.alert-text");
 			padding: 0 .4em;
 			position: absolute;
-			right: 10px;
```
Comment (sanlucca) [inline packages/scss/src/components/_nav-side.scss:286]: In compact mod, this new should be below the text, centered and it should extend the "selection" box :)

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1022

File: packages/scss/src/components/_nav-side.scss (hunk 1 above)
Comment (RipripKipkip) [reply packages/scss/src/components/_nav-side.scss:286]: image

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1022

**Pull request merged**